src/components/app/Dog.md

```diff
@@ -0,0 +1,17 @@
+```jsx
+<Dog
+  id="shetland-sheepdog"
+  breed="Shetland Sheepdog"
+  kidFriendly={5}
```

Comment:
These are missing in propTypes. I think this data should only require the id and then be fetched from the DB.

Reply:
The rendering component shouldn't fetch any data, and all the data is already fetched by this time.